<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ACIAView</class>
 <widget class="QDialog" name="ACIAView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>CS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitsView" name="chipSelect" native="true">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="RegisterView" name="statusRegister" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TX:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="txData">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitsView" name="txFlag" native="true">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>RX:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="rxData">
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="BitsView" name="rxFlag" native="true">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="RegisterView" name="controlRegister" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="RegisterView" name="commandRegister" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>10</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Console" name="console"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RegisterView</class>
   <extends>QWidget</extends>
   <header>views/RegisterView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BitsView</class>
   <extends>QWidget</extends>
   <header>views/BitsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Console</class>
   <extends>QPlainTextEdit</extends>
   <header>views/Console.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
